docs/data-access/daco/renew-and-close.md

```diff
@@ -5,52 +5,63 @@ sidebar_label: Attesting, Renewing, & Closing
+
+Following your attestation, the application status will revert to the approved state. If access was previously paused, it will be reinstated for your project team.
 
 ## Renewing an Application
 
-The DACO approval expires after 2 years of access. An email reminder will be sent to the applicant 90 days and 45 days prior to the access expiry date as well as on the date of expiry. If the project team is to remain active, the applicant may wish to renew beyond the 2 years. The option to renew the application will be available 90 days before the application expires, and up to 90 days after it expires.
+The DACO approval expires after 2 years of access. An email reminder will be sent to the applicant 90 days and 45 days prior to the access expiry date, as well as on the date of expiry. If the project team is to remain active, the applicant may wish to renew beyond the 2 years. The option to renew the application will be available 90 days before the application expires, and up to 90 days after it expires.
 
 ### How to Renew an Application
 
 1. When the applicant receives the email reminder that the application is about to expire (or has expired), they can login and click the **Renew** button.
+
    ![DACO renew button for an expired application](/assets/data-access/daco-renew-button.png)
-1. At this time, the application will be unlocked and you can make any changes (if necessary) to the currently approved application, agree to the Data Access Agreement and Appendices again, and re-sign and submit the renewal application.
+
+2. Upon clicking the **Renew** button, a new, separate draft renewal application is created. This draft is linked to the original application for reference purposes. The applicant must complete this new draft application for renewal. The original application cannot be edited directly.
+
    ![DACO renewal application](/assets/data-access/daco-renewal-application.png)
-1. The DACO Administrator will be notified that a renewal application is ready for review. If there are no issues and/or the application remains the same as the previous version, the project team should be quickly approved for an additional 2 years.
+
+3. The DACO Administrator will be notified that a renewal application is ready for review. If there are no issues and/or the application remains consistent with the previous version, the project team should be quickly approved for an additional 2 years.
 
 ### Expired and Non-renewable
 
-A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not received within 90 days of the application's expiry date, it will be considered closed permanently.
+A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not submitted within 90 days of the application's expiry date, it will be closed permanently.
```

Comment:
would slightly modify: `If a renewal is not submitted within 90 days of the original application's expiry date, the renewal will be closed permanently.`